css/theme/marcosh.css

```diff
@@ -56,8 +56,8 @@ pre code {
 .reveal footer img {
   position: fixed;
   bottom: 10px;
-  height: 70px;
-  width: 80px;
+  height: 140px;
```

Comment:
making it this big interferes with the slides, since it overlaps with them

Reply:
This depends on the screen size/resolution. I scaled it by 2, maybe scale it by 1.5?